graffiti/storage/elasticsearch/client.go

```diff
@@ -90,6 +94,30 @@ type Client struct {
 	masterElection etcd.MasterElection
 }
 
+// TraceLogger implements the oliviere/elastic Logger interface to be used with trace messages
+type TraceLogger struct{}
+
+// Printf sends elastic trace messages to skydive logger Debug
+func (l TraceLogger) Printf(format string, v ...interface{}) {
+	logging.GetLogger().Debugf(format, v)
```

Comment:
This generates huge lot entry, especially it dumps all the thing we insert. It seems to slow down the CI job to have it killed after a timeout. I would rather use Tracef here

Reply:
But ``Tracef`` does not exist, right?
https://github.com/skydive-project/skydive/blob/f4de9c5201a2cc4b5b98a448aedb07e370adaf91/graffiti/logging/logging.go#L31

Reply:
You're right, so I would make it optional, what about adding a config parameter to enable the tracing ?